### ISSUE TITLE
Still alive?

### ISSUE THREAD

**Issue by sefirosweb** [rase-/socket.io-php-emitter#36]
Hello guys seems this proyect is dead?

Seems is not working for Socket.io 4 and no updates / answers

Why socket.io officialsite indicates this project?
https://socket.io/docs/v4/redis-adapter/#Emitter

**Comment by robgonnella** [comment]
@rase- I just ran into this issue myself and I came up with a solution that is working for me on v4.5.1.

**Comment by alielmajdaoui** [comment]
@robgonnella thanks for the updates.

The author of this repo should leave a notice that it is no longer maintained and link to another one. Redis docs should be updated too.

**Comment by rayyzhong** [comment]
> @robgonnella thanks for the updates.
> 
> The author of this repo should leave a notice that it is no longer maintained and link to another one. Redis docs should be updated too.

Exactly!!!  Now it is 2023! Alive, or dead, by no mean for any ambiguity.
